### PR TITLE
Add metaphlan tool suite

### PR DESCRIPTION
**Pull request by bebatut**
Should I add the data manager there too?

### CODE REVIEW

**Comment by bgruening** [comment]
Yes please add data manager as well and remove the older one, also from the .lock file please.

File: tools_iuc.yaml [hunk 1]
```diff
@@ -3391,3 +3391,26 @@ tools:
   - name: pe_histogram
     owner: iuc
     tool_panel_section_label: Graph/Display Data
+  
+  - name: metaphlan
```
Comment (bgruening) [inline tools_iuc.yaml:3395]: I have 12 new repos for human related tools ...

Comment (bebatut) [reply tools_iuc.yaml:3395]: I wanted to add with a new PR :)

Comment (bgruening) [reply tools_iuc.yaml:3395]: a 14 ...

**Pull request merged**